autodispose/src/test/java/com/uber/autodispose/AutoDisposeObserverTest.java

```diff
@@ -154,7 +162,7 @@
     lifecycle.onNext(1);
     source.onNext(2);
 
-    assertThat(source.hasObservers()).isTrue();
+    assertThat(source.hasObservers()).isTrue();Å
```

Comment:
wat.gif

Reply:
I believe this was from you trying to use my keyboard.